hooks/aci_utils.py

```diff
@@ -6,16 +6,10 @@
 import sys
 from itertools import chain
 import yaml
-import pdb
+import os
 
-ACI_PACKAGES = [
-   'python-apicapi',
-   'group-based-policy',
-   'python-group-based-policy-client',
-   'neutron-opflex-agent',
-   'aci-integration-module',
-   'acitoolkit',
-]
+os.environ["LANG"]="C.UTF-8"
```

Comment:
Did we confirm that we still need these? I remember we saw an issue, but I wasn't sure if these were still needed after all the changes.